libs/wingc/src/jsify.rs

```diff
@@ -1325,7 +1326,10 @@ fn get_public_symbols(scope: &Scope) -> Vec<Symbol> {
 }
 
 fn inflight_filename(class: &AstClass) -> String {
-	format!("./inflight.{}.js", class.name.name)
+	let mut hasher = Sha1::new();
+	hasher.update(&class.name.span.file_id);
```

Comment:
Is it promised that file_id is not a full path? If it is well have a determinism issue, no?

Maybe a simple counter can be sufficient

Reply:
It is promised that it's not a full path, with the exception of cases where you provided a full path to `wing compile <file>`